nss-evening-cohort-26/FE-Hackathon-music-streaming-project
# Read playlist/bg

## Description
Playlists is shown when Playlists is clicked in the NavBar. A component was created for the layout of the Playlist card and the playlist.js route was updated with functionality and api call to show all playlists.

## Related Issue
#6 

## Motivation and Context
The motivation behind this is we want users to see all their playlists they have (if any) when the user is logged in.

## How Can This Be Tested?
git fetch origin readPlaylist/bg
git checkout readPlaylist/bg
npm run dev

**Make sure** to instal react icons to see icons for view, edit and delete on playlist card if you have any: **_npm install react-icons_**

## Screenshots (if appropriate):

## Types of changes
